initials.c

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
```

Comment:
Great job! Your code runs and you followed the guidelines - the only thing I'd add is a comment with your name and purpose of this code since it's good practice to do so. Great job overall 👍